src/Analyzers/CSharp/Tests/ConvertSwitchStatementToExpression/ConvertSwitchStatementToExpressionTests.cs

```diff
@@ -1966,5 +1966,46 @@ public async Task TopLevelStatement_FollowedWithThrow()
 
             await test.RunAsync();
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsConvertSwitchStatementToExpression)]
+        [WorkItem(48006, "https://github.com/dotnet/roslyn/issues/48006")]
+        public async Task TestOnMultiCaseSection_String_CSharp9()
+        {
+            var testCode = @"
+class Program
+{
+    bool M(string s)
+    {
+        [|switch|] (s)
+        {
+	        case ""Last"":
```

Comment:
The formatting here looks a little off, although it looks like it was that way in the original too, so not sure if it's a big deal?

Reply:
Oops nevermind, it looks fine now for some reason, guess something might have just been off earlier on my end.

Reply:
Oops nevermind, it looks fine now for some reason, guess something might have just been off earlier on my end.